.travis.yml

```diff
@@ -54,6 +54,7 @@ install:
   - |
     if [[ "$INSTALL_WORDPRESS" = "true" ]]; then
       echo '{ "config": { "SCRIPT_DEBUG": false, "WP_PHP_BINARY": "php" } }' > .wp-env.override.json
+      chmod -R 767 ./ # TODO: Possibly integrate in wp-env
```

Comment:
should probably make this go back to doing ./build. doing all of cwd took a long time for me locally. (plus we should figure out why we need this)

Reply:
Do you know why this "chmod" is needed?

Reply:
Yes, some of the tests needed filesystem write access, which was not working until we added this. Did you end up needing it during the switch to GitHub workflows?